data/json/items/tool_armor.json

```
@@ -2763,7 +2763,7 @@
   {
     "id": "scuba_tank",
     "type": "TOOL_ARMOR",
-    "category": "other",
+    "category": "tools_entry",
```

Scuba tank seems also a lil goofy as an entry tool.
```suggestion
    "category": "tool",
```